cms/toolbar/items.py

```diff
@@ -236,10 +243,10 @@ def get_context(self):
 class Menu(SubMenu):
     sub_level = False
 
-    def get_or_create_menu(self, key, verbose_name, side=LEFT, position=None):
+    def get_or_create_menu(self, key, verbose_name, disabled=False, side=LEFT, position=None):
```

Comment:
Please place the kwarg at the end to avoid any backwards incompatible issues where positional arguments are used